test/binary-op.civet

```diff
@@ -827,3 +827,79 @@ describe "custom identifier infix operators", ->
     foo(a+b, c*d) < x
     a || foo(b, c) && d
   """
+
+  describe "custom precedence", ->
+    testCase """
+      same as symbol
+      ---
+      operator mul same (*) (a, b)
+        a*b
+      a+b*c mul d*e+f
+      ---
+      function mul (a, b) {
+        return a*b
+      }
+      a+mul(b*c, d)*e+f
```

Comment:
If `mul` is the same precedence as `*` shouldn't it be:

`a+b*mul(c, d)*e+f`

Reply:
No, it's acting left-associative with `*`, so `b*c` goes first.